parser/proto.y

```diff
@@ -33,19 +33,18 @@ import (
 	ooElements   []ast.OneofElement
 	ext          *ast.ExtensionRangeNode
 	resvd        *ast.ReservedNode
+	resvdN       nodeWithEmptyDecls[*ast.ReservedNode]
```

Comment:
The naming choice here is a little confusing. At the moment, it looks like the real difference between these (other than types) is that one is used for messages and one for enums (at least for now).

So maybe name them `msgResvd` and `enumResvd`?

Reply:
I am going to update msgResvd to use reservdN, delete reservd and rename reservdN to reserved. I was trying to minimize the amount to churn that would take, and keep the diff small.

Reply:
Added a TODO with this info